target/linux/x86/config-5.10

```diff
@@ -479,28 +370,28 @@ CONFIG_X86=y
 CONFIG_X86_32=y
 # CONFIG_X86_32_IRIS is not set
 CONFIG_X86_32_LAZY_GS=y
-CONFIG_X86_ALIGNMENT_16=y
 # CONFIG_X86_ANCIENT_MCE is not set
 # CONFIG_X86_CHECK_BIOS_CORRUPTION is not set
+CONFIG_X86_CMOV=y
```

Comment:
This will break x86/legacy target for some older CPUs not supporting this. When refreshing this config You should use the lowest denominator which is the legacy subtarget.

Reply:
Refreshing configs is an automated process that avoids manual tweaking in the interest of keeping a maintainable, repeatable config. It appears in this case a few things will need manual changes, but at least these can be isolated in a separate commit.

Please note any other legacy config settings since you're familiar with them. CONFIG_X86_F00F_BUG seems like another?

Reply:
Looking more closely, there should be no issue here. These config options are non-prompt and depend on the selected CPU:
```
config X86_CMOV
	def_bool y
	depends on (MK8 || MK7 || MCORE2 || MPENTIUM4 || MPENTIUMM || MPENTIUMIII || MPENTIUMII || M686 || MVIAC3_2 || MVIAC7 || MCRUSOE || MEFFICEON || X86_64 || MATOM || MGEODE_LX)
```

Reply:
> Refreshing configs is an automated process that avoids manual tweaking in the interest of keeping a maintainable, repeatable config. It appears in this case a few things will need manual changes, but at least these can be isolated in a separate commit.

This is a semi automated process, the subtarget will always be selected and it'll influence the resulting config for whole target. If You' d selected legacy subtarget there would be no need to manually adjust the problematic symbols.


> Please note any other legacy config settings since you're familiar with them. CONFIG_X86_F00F_BUG seems like another?

Yes that too, but I don't think the Pentiums requiring that are used anymore. BTW that's automatic selection.